PyRDF/backend/Executor.py

```diff
@@ -12,7 +12,7 @@ def check_env(self):
             return "local"
         return env
 
-    def execute(self, generator):
+    def execute(self, generator, env="local", conf={}):
```

Comment:
This is more a general comment, but sometimes you include changes in a commit that do not have to do with the commit message. Like this change and the ones below are related to the backend selection and to the synchronization of the final results, that is three different topics in the same commit, which makes it harder for us to review. Please take this into account in future commits!

Reply:
Yes @etejedor . I'd commited files after fixing multiple issues. I shouldve pushed a commit per issue. This is a big mistake. I'll make it a point next time to open a PR per issue :)